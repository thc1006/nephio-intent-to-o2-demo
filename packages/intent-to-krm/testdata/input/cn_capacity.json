{
  "intentExpectationId": "cn-cap-001", 
  "intentExpectationType": "ServiceCapacity",
  "intentExpectationContext": {
    "contextAttribute": "networkSlice",
    "contextCondition": "EQUAL",
    "contextValueRange": ["slice-urllc"]
  },
  "intentExpectationTarget": {
    "targetAttribute": "latency",
    "targetCondition": "LESS_THAN",
    "targetValue": "1ms"
  },
  "intentExpectationObject": {
    "objectType": "CoreNetwork",
    "objectInstance": "5gc-upf-001"
  }
}