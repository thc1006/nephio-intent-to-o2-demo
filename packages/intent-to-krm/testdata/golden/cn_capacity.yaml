apiVersion: v1
kind: ConfigMap
metadata:
  name: expectation-cn-cap-001
  namespace: intent-to-krm
  labels:
    expectation.nephio.org/id: cn-cap-001
    expectation.nephio.org/object-type: CoreNetwork
    expectation.nephio.org/type: ServiceCapacity
  annotations:
    expectation.nephio.org/context-attribute: networkSlice
    expectation.nephio.org/context-condition: EQUAL
    expectation.nephio.org/target-attribute: latency
    expectation.nephio.org/target-condition: LESS_THAN
    expectation.nephio.org/target-value: 1ms
data:
  expectation.json: |-
    {
      "intentExpectationId": "cn-cap-001",
      "intentExpectationType": "ServiceCapacity",
      "intentExpectationContext": {
        "contextAttribute": "networkSlice",
        "contextCondition": "EQUAL",
        "contextValueRange": ["slice-urllc"]
      },
      "intentExpectationTarget": {
        "targetAttribute": "latency",
        "targetCondition": "LESS_THAN",
        "targetValue": "1ms"
      },
      "intentExpectationObject": {
        "objectType": "CoreNetwork",
        "objectInstance": "5gc-upf-001"
      }
    }
---
apiVersion: cn.nephio.org/v1alpha1
kind: CNBundle
metadata:
  name: cn-bundle-cn-cap-001
  namespace: intent-to-krm
  labels:
    expectation.nephio.org/id: cn-cap-001
    expectation.nephio.org/type: ServiceCapacity
spec:
  expectationId: cn-cap-001
  expectationType: ServiceCapacity
  networkSlices:
    - slice-urllc
  capacity:
    latency:
      condition: LESS_THAN
      value: 1ms
  objectInstance: 5gc-upf-001